### PR TITLE
Adding real data file

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds the current version of the feed ingestion service YAML file for POC purposes.

## Affected pages

https://adobedocs.github.io/commerce-services/openapi/

### CODE REVIEW

**Review by jhadobe — CHANGES_REQUESTED** [review]
Generates fine locally. I would suggest removing the `openapi.yaml` file if it is no longer needed.

Also, are we still intentionally not adding this link to the side navigation for the time being?

**Comment by dobooth** [comment]
Done. I am just testing the file and the setup. Not TOC because it is not going public yet. Just getting the basics settled ahead of time.

**Review by keharper — CHANGES_REQUESTED** [review]
This cannot be checked into `main`. Create a branch called something like `feed-ingestion-develop`. More comments forthcoming.

**Comment by dobooth** [comment]
Done. I trust we can set the branch where the github docs are pulled from.

**Comment by keharper** [comment]
Change the path to `src/pages/feed-ingestion/index.md`. Eventually, the content of that file will be replaced with some conceptual/setup/use case info, and the REST reference will have a permanent home. Until then, the URL will make sense to your beta customers. We'll also theoretically be able to create autogenerated REST docs for Recommendations and Catalog Service and have a consistent DX.

File: src/pages/feed-ingestion/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 title: Adobe Commerce
 description: This is an OpenAPI test.
-openAPISpec: /openapi.yaml
+openAPISpec: /spec.yaml
```
Comment (keharper) [inline src/pages/feed-ingestion/index.md:4]:
```suggestion
openAPISpec: /feed-ingestion.yaml
```

File: src/pages/feed-ingestion/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
 title: Adobe Commerce
```
Comment (keharper) [inline src/pages/feed-ingestion/index.md:2]:
```suggestion
title: Adobe Commerce Feed Ingestion REST endpoints
```

File: src/pages/feed-ingestion/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 description: This is an OpenAPI test.
```
Comment (keharper) [inline src/pages/feed-ingestion/index.md:3]:
```suggestion
description: Reference information for Adobe Commerce Feed Ingestion REST endpoints.
```

**Review by keharper — APPROVED** [review]

**Pull request merged**